### PR TITLE
adjust MTV assessment to new results schema

### PR DESCRIPTION
**Pull request by ydayagi**
FLPATH-1734
https://issues.redhat.com/browse/FLPATH-1734

### CODE REVIEW

**Comment by ydayagi** [comment]
@masayag @ElaiShalevRH I have lgtm. i need also approve :-)

**Comment by gciavarrini** [comment]
The jira says 
> MTV should be fixed to put the next workflow suggestions in the new results structure
> In addition any other related links and info can be put there as well. 

Can you please explain where "other related links and info" are/can be set?

**Comment by ydayagi** [comment]
> The jira says
> 
> > MTV should be fixed to put the next workflow suggestions in the new results structure
> > In addition any other related links and info can be put there as well.
> 
> Can you please explain where "other related links and info" are/can be set?

looking at gabriel's example it seems that this part did not change. so old code still works.

File: mtv-plan/mtv.sw.yaml [hunk 1]
```diff
@@ -119,5 +121,5 @@ states:
               topic: "MTV plan assessment workflow"
               severity: "normal"
     stateDataFilter:
-      output: '{workflowOptions: {migrationOptions: [{id: "mtv-migration", name: "MTV migration workflow"}]}, assessmentResult: {passed: true, message: "plan is ready for migration"}}'
+      output: '{workflowOptions: {migrationOptions: [{id: "mtv-migration", name: "MTV migration workflow"}]}}'
```
Comment (gabriel-farache) [inline mtv-plan/mtv.sw.yaml:124]: this output is not good, it does not matches the schema defined by `mtv-plan/schemas/workflow-output-schema.json`:

https://github.com/parodos-dev/serverless-workflows/pull/439/files#diff-a1e81bb077285ef5ac194f923417a1ae412c439357a9f3d725d909d9a84b2187R106 is good

You should have something similar here:
```
      output: '{result: {completedWith: "success", message: "MTV plan successful", "nextWorkflows": [{id: "mtv-migration", name: "MTV migration workflow"}], "outputs": [] }}'
```

**Review by gabriel-farache — CHANGES_REQUESTED** [review]
The output of the successful state is wrong

File: mtv-plan/mtv.sw.yaml (hunk 1 above)
Comment (ydayagi) [reply mtv-plan/mtv.sw.yaml]: fixed

**Comment by ydayagi** [comment]
> The output of the successful state is wrong

@gabriel-farache please take a look at my latest changes

**Review by gabriel-farache — APPROVED** [review]

**Pull request merged**